Source/WebCore/platform/graphics/cocoa/IOSurface.mm

```diff
@@ -441,6 +462,10 @@ static IntSize computeMaximumSurfaceSize()
         break;
     case Format::RGBA:
         break;
+#if HAVE(HDR_SUPPORT)
+    case Format::RGBA16F:
+        break;
```

Comment:
Would it make sense here to copy what's in the "RGB10" case above? I.e.:
```
        bitsPerComponent = 16;
        bitmapInfo = static_cast<CGBitmapInfo>(kCGImageAlphaPremultipliedLast) | static_cast<CGBitmapInfo>(kCGBitmapByteOrder16Host) | static_cast<CGBitmapInfo>(kCGBitmapFloatComponents);
```

Reply:
Yes. This makes sense.